src/python/pants/backend/helm/subsystems/post_renderer.py

```diff
@@ -48,7 +48,7 @@ class HelmPostRendererSubsystem(PythonToolRequirementsBase):
     ]
 
     register_interpreter_constraints = True
-    default_interpreter_constraints = ["CPython>=3.7,<3.10"]
```

Comment:
The 3.10 here is incongruent but pre-existing, intend to clean it up after this one lands.